queries/buildings.jinja2

```diff
@@ -57,10 +58,11 @@ SELECT
     NULL AS area,
     osm_id AS __id__,
     NULL AS "building:part",
+    "address" AS kind,
     NULL AS building,
-    amenity,
-    shop,
-    tourism,
+    NULL AS amenity,
```

Comment:
The `address` nature is important for addr points, the amenity, shop, tourism is not.